app/ollama_welcome.ps1

```diff
@@ -0,0 +1,10 @@
+# TODO - consider ANSI colors and maybe ASCII art...
+write-host ""
+write-host "`tWelcome to Ollama!"
```

Comment:
I think we can do without this `t`
```suggestion
write-host "`Welcome to Ollama!"
```

The newlines/before after give it a nice padding